gunicorn/glogging.py

```diff
@@ -81,7 +102,7 @@
             "generic": {
                 "format": "%(asctime)s [%(process)d] [%(levelname)s] %(message)s",
                 "datefmt": "[%Y-%m-%d %H:%M:%S %z]",
-                "class": "logging.Formatter"
+                "class": "Formatter"
```

Comment:
Doesn't this have to be the full `gunicorn.glogging.Formatter`?

Reply:
@wking thanks, I changed it to `gunicorn.glogging.Formatter`